katsdpcontroller/product_controller.py

```diff
@@ -998,12 +998,17 @@ def __del__(self) -> None:
         finally:
             init_time = capture_block.state_time[CaptureBlockState.CAPTURING]
             done_time = capture_block.state_time[CaptureBlockState.BURNDOWN]
-            obs_time = done_time - init_time
+            observation_time = done_time - init_time
             postprocessing_time = time.time() - done_time
             POSTPROCESSING_TIME.observe(postprocessing_time)
+            logger.info('Capture block %s postprocessing finished in %.3fs',
```

Comment:
Might be worth including the observation time here for nice quick comparisons...

Reply:
Done.